ros2_control_test_nodes/ros2_control_test_nodes/publisher_forward_position_controller.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
         self.declare_parameter("goal_names", ["pos1", "pos2"])
 
         # Read parameters
-        controller_name = self.get_parameter("controller_name").value
+        # controller_name = self.get_parameter("controller_name").value
```

Comment:
bam! what is this sir?

Reply:
I wanted to push “illegal” stuff through.

The variable is not actually used, so linter is complaining. 

We have to anyway remove this package since it is moved to the controllers.